python/ray/data/read_api.py

```diff
@@ -105,6 +105,8 @@ def from_items(
     items: List[Any],
     *,
     parallelism: int = -1,
+    concurrency: Optional[int] = None,
+    override_num_blocks: Optional[int] = None,
```

Comment:
Don't think `override_num_blocks` is used in this implementation? Also, could we raise a deprecation warning if the user specifies `parallelism`?

Reply:
Yes I will add warning. thanks.

Reply:
> Don't think override_num_blocks is used in this implementation?

It is used to determine how many objects are bundled as one input for the next op.